lib/swagger-client.js

```diff
@@ -1478,9 +1481,36 @@ Operation.prototype.encodePathParam = function(pathParam) {
   }
 };
 
-var Model = function(name, definition) {
+var Model = function(name, definition, allDefinitions) {
   this.name = name;
   this.definition = definition || {};
+  if(definition.allOf) {
+    var resolve = function(definition, updatedDefinition, ignoreModels) {
+      if (!definition)
+        return;
+      
+      if (definition.allOf) {
+        for (var i=0,len=definition.allOf.length;i<len;i++) {
+          resolve(definition.allOf[i], updatedDefinition, ignoreModels);
+        }
+      } else if (definition.$ref) {
+        var simpleName = simpleRef(definition.$ref);
+        if (typeof ignoreModels[simpleName] !== true) {
```

Comment:
Whoops!  Sorry about that, the "typeof" was supposed to be removed.  I'll get that fixed.